src/box/iproto.cc

```diff
@@ -3107,6 +3221,12 @@ struct iproto_cfg_msg: public cbus_call_msg
 		struct evio_service *binary;
 		/** New iproto max message count. */
 		int iproto_msg_max;
+		struct {
+			/** Overridden request type. */
+			uint32_t req_type;
+			/** Whether the request handler is set or reset. */
+			bool is_set;
```

Comment:
Nit: Better name the struct so that it's clear what it's for, e.g.:

```C
struct {
    uint32_t type;
    bool is_set;
} override;
```

Reply:
Fixed.